app/controllers/admin/application_controller.rb

```diff
@@ -14,7 +14,7 @@ class Admin::ApplicationController < ApplicationController
   def auto_complete
     @query = params[:auto_complete_query]
     @auto_complete = klass.text_search(@query).limit(10)
-    render :partial => 'auto_complete'
+    render partial: 'auto_complete'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.